static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-opt.js

```diff
@@ -0,0 +1,49 @@
+export const signInGateCopyOpt = {
+    id: 'SignInGateCopyOpt',
+    start: '2021-02-24',
+    expiry: '2021-12-01',
+    author: 'rebecca-thompson',
+    description:
+        'Compare 6 different gate copy changes, on 3rd page view of the day, on simple article templates, with higher priority over banners and epi',
+    audience: 0.2,
```

Comment:
Shouldn't this be 0.3? Or is there something I'm missing?

Reply:
the last 10% is for the control (no gate) with a 0.9 offset and 0.1 audience